systemtest/src/test/resources/oauth2/prepare_pg_operator.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+PGO_INSTANCE_NAMESPACE=$1
+PGO_VERSION=$2
+PGO_VERSION="${PGO_VERSION:=main}"
+PGO_BASEDIR=/tmp/pgo/poe
+
+echo $PGO_VERSION
+
+echo "[INFO] $(date -u +"%Y-%m-%d %H:%M:%S") Deploy Postgres Operator"
+[[ -d "${PGO_BASEDIR}" ]] && rm -rf "${PGO_BASEDIR}"
+mkdir -p "${PGO_BASEDIR}"
+git clone https://github.com/CrunchyData/postgres-operator-examples.git --branch ${PGO_VERSION} "${PGO_BASEDIR}"
+for file in $(grep -rin "namespace: postgres-operator" "${PGO_BASEDIR}" | cut -d ":" -f 1); do sed -i "s/namespace: .*/namespace: ${PGO_INSTANCE_NAMESPACE}/" "${file}"; done
+
+kubectl create namespace "${PGO_INSTANCE_NAMESPACE}"
+kubectl apply --server-side -k "${PGO_BASEDIR}/kustomize/install/default"
+sleep 10
+
+echo "[INFO] $(date -u +"%Y-%m-%d %H:%M:%S") Wait for Postgres Operator readiness"
+kubectl wait deployment/pgo --for=condition=available --timeout=90s -n ${PGO_INSTANCE_NAMESPACE}
+kubectl wait deployment/pgo-upgrade --for=condition=available --timeout=90s -n ${PGO_INSTANCE_NAMESPACE}
```

Comment:
Did you try these deployments on Minikube with for instance `6900MB RAM a 2vCPUs`? I wonder if the 90s is enough for that...

Reply:
Not yet, I will run AZP regression as you suggested.